module3/homework/advanced_calculator/test.cpp

```diff
@@ -84,7 +84,7 @@ TEST(advancedCalculatorTest, ShouldPower) {
     EXPECT_TRUE(cmp(result, 0.000976562));
 }
 
-TEST(advancedCalculatorTest, ShouldCalculateSqrt) {
+TEST(advancedCalculatorTest, ShouldCalculateRoot) {
```

Comment:
Could you explain me why root is more appropriate?

Reply:
because sqrt means square root. So sqrt(4) = 2. But when you look at test you can see 3rd root, 4th root etc 
`ASSERT_EQ(process("1024$4", &result), ErrorCode::OK);`
 `ASSERT_EQ(process("225 $3", &result), ErrorCode::OK);` 
and these are not square roots

Reply:
I didn't know that. Thanks :)